test/e2e/scheduling/preemption.go

```diff
@@ -138,8 +127,19 @@ var _ = SIGDescribe("SchedulerPreemption [Serial]", func() {
 				PriorityClassName: priorityName,
 				Resources: &v1.ResourceRequirements{
 					Requests: podRes,
+					Limits:   podRes,
+				},
+				Affinity: &v1.Affinity{
+					NodeAffinity: &v1.NodeAffinity{
+						RequiredDuringSchedulingIgnoredDuringExecution: &v1.NodeSelector{
+							NodeSelectorTerms: []v1.NodeSelectorTerm{
+								{MatchExpressions: []v1.NodeSelectorRequirement{
+									{Key: "kubernetes.io/hostname", Operator: v1.NodeSelectorOpIn, Values: []string{node.Labels["kubernetes.io/hostname"]}},
```

Comment:
It'd be more robust to replace this with `MatchFields`, as the node may not have "kubernetes.io/hostname" label: (applies elsewhere)

```go
MatchFields: []v1.NodeSelectorRequirement{
    {Key: "metadata.name", Operator: v1.NodeSelectorOpNotIn, Values: []string{node.Name}},
},
```

Reply:
> v1.NodeSelectorOpNotIn

I'm assuming you mean OpIn, updated

Reply:
ah, yes..